angular/core/components/explore_page/explore_page_controller.coffee

```diff
@@ -0,0 +1,43 @@
+angular.module('loomioApp').controller 'ExplorePageController', (Records, $rootScope, $timeout, AppConfig, LoadingService) ->
+
+  @groupIds = []
+  @perPage = AppConfig.pageSize.exploreGroups
+  @canLoadMoreGroups = true
+  @query = ""
+  $timeout -> document.querySelector('#search-field').focus()
+
+  @groups = =>
+    Records.groups.find(@groupIds)
+
+  @search = =>
+    if @query
+      @groupIds = []
+    Records.groups.fetchExploreGroups(@query, {from: @groupIds.length, per: @perPage}).then (object) =>
+      @groupIds = @groupIds.concat(_.map object.groups, (group) -> group.id)
+      if (object.groups or []).length < @perPage
+        @canLoadMoreGroups = false
+
+  LoadingService.applyLoadingFunction @, 'search'
+  @search()
+
+  @groupCover = (group) ->
+    { 'background-image': "url(#{group.coverUrl()})" }
+
+  @groupDescription = (group) ->
+    _.trunc group.description, 100 if group.description
+
+  @showMessage = ->
+    !@searching &&
+    @query &&
+    @groups().length > 0
+
+  @searchResultsMessage = ->
+    if @groups().length == 1
+      'explore_page.single_search_result'
+    else
+      'explore_page.multiple_search_results'
+
+  @noResultsFound = ->
+    !@searching && (@groups().length < @perPage || !@canLoadMoreGroups)
```

Comment:
Yeah, the message says "Can't find the group you're looking for?...", and, according to the design, it is displayed when there are no results or when you've gotten to the end of the results.